mtgjson5/providers/tcgplayer.py

```diff
@@ -171,6 +240,61 @@ def generate_tcgplayer_to_mtgjson_map(
     return dump_map
 
 
+def get_tcgplayer_buylist_prices_map(
+    group_id_and_name: Tuple[str, str]
```

Comment:
Also, why a tuple? This is prime for `group_id: str, group_name: str` which is cleaner

Reply:
^ That goes for everywhere you used a tuple btw